### DISCUSSION TITLE
Splash Image Persists After Application Launch

### DISCUSSION THREAD

**Discussion by haqueshenas** [pyinstaller/pyinstaller#8166] [Help] [answered]
I used auto-py-to-exe to convert my Python code into an .exe file and configured the splash image in the Advanced section. However, when running the converted .exe file, I noticed that the image persists on the screen until the application is closed. I'm seeking guidance on resolving this issue. Thank you!

**Comment by bwoodsend** [comment, marked as answer]
Are you calling [pyi_splash.close()](https://pyinstaller.org/en/stable/usage.html?highlight=splash#the-pyi-splash-module) near the start of your program?

**Reply by haqueshenas** [reply]
Thanks. I added the below lines to the first part of my code, and the problem was solved:

try:
    import pyi_splash
    pyi_splash.close()
except:
    pass
